src/components/Tabs/Tabs.jsx

```diff
@@ -16,7 +16,7 @@ export const Tabs = ({ className, variant = "default", value, onChange, children
 
     return (
         <>
-            <nav className={clsx("ui-tabs", "flex overflow-x-auto", variants[variant], className)} {...rest}>
+            <nav className={clsx("ui-tabs", "flex", variants[variant], className)} {...rest}>
```

Comment:
We need this. The fix is to add `<ColumnLayout.Fill>` around the page content in x2-seller